emscripten-requirements.txt

```diff
@@ -1,3 +1,3 @@
-pytest-pyodide==0.54.0
-pyodide-build==0.24.1
-webdriver-manager==4.0.1
\ No newline at end of file
+pytest-pyodide==0.58.2
+pyodide-build==0.26.2
```

Comment:
With the pyodide-build unvendoring, do we need a pin here @ryanking13? Should it pin latest?

Reply:
You don't need to set the pyodide-build version equal to the target Pyodide version anymore, but I think it is not a bad thing to pin the version for reproducible build.

Reply:
Actually, thinking about it, I don't need pyodide-build any more, not sure that should be in really. Will fix.